### ISSUE TITLE
准确率

### ISSUE THREAD

**Issue by yida88** [KaiyangZhou/deep-person-reid#157]
您好，请问最近改的代码与改之前的代码在准率上有提升吗

**Comment by KaiyangZhou** [comment]
those changes were not made for improving the accuracy

**Comment by yida88** [comment]
不好意思，我再重述下我的问题。在没改之前的训练命令是python train_imgreid_xent.py \
-s market1501 \ # source dataset for training
-t market1501 \ # target dataset for test
--height 256 \ # image height
--width 128 \ # image width
--optim amsgrad \ # optimizer
--label-smooth \ # label smoothing regularizer
--lr 0.0003 \ # learning rate
--max-epoch 60 \ # maximum epoch to run
--stepsize 20 40 \ # stepsize for learning rate decay
--train-batch-size 32 \
--test-batch-size 100 \
-a resnet50 \ # network architecture
--save-dir log/resnet50-market-xent \ # where to save the log and models
--gpu-devices 0 \ # gpu device index
改之后的训练命令是：python main.py \
--root path/to/reid-data \
--app image \
--loss softmax \
--label-smooth \
-s market1501 \
-a resnet50 \
--optim adam \
--lr 0.0003 \
--max-epoch 60 \
--stepsize 20 40 \
--batch-size 32 \
--save-dir log/resnet50-market-softmax \
--gpu-devices 0
这个改变对准确率没影响吧，还有改之后的再加上三元组损失函数的训练命令是？
谢谢
谢

**Comment by KaiyangZhou** [comment]
The code change will not make a difference to the accuracy.

For triplet loss training, please have a look at the argument list https://github.com/KaiyangZhou/deep-person-reid/blob/master/scripts/default_parser.py

**Comment by yida88** [comment]
非常感谢
